flocker/acceptance/test_postgres.py

```diff
@@ -114,38 +114,47 @@ def setUp(self, cluster):
                 [u"applications", POSTGRES_APPLICATION_NAME, u"ports", 0,
```

Comment:
I think most of the changes here are unnecessary, since you can just return Deferred from `setUp`.